test/Parse/diagnostic_missing_func_keyword.swift

```diff
@@ -3,7 +3,7 @@
 // https://github.com/apple/swift/issues/52877
 
 protocol Brew { // expected-note {{in declaration of 'Brew'}}
-  tripel() -> Int // expected-error {{expected 'func' keyword in instance method declaration}} {{3-3=func }}
+  triple() -> Int // expected-error {{expected 'func' keyword in instance method declaration}} {{3-3=func }}
 
   quadrupel: Int { get } // expected-error {{expected 'var' keyword in property declaration}} {{3-3=var }}
```

Comment:
I would leave these alone. Infusing what is likely a silly random name with meaning by correcting it to the closest legit word is as likely to lead to confusion.

Reply:
reverted

Reply:
Still there.

Reply:
Fixed now, darn, must have missed a git add....

Reply:
Just adding that "tripel" is actually [a term of art in beer](https://en.wikipedia.org/wiki/Tripel) related to "quadrupel", so Anthony is almost certainly correct that this was intentional.